src/platforms/python/common/configuration/integrations/loguru.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Loguru
+description: Learn about using Sentry with loguru.
+sidebar_order: 80
+---
+
+The [Loguru](https://github.com/Delgan/loguru#readme) integration lets you capture log messages and send them to Sentry.
+
+Most of the work is handled by the [`logging`](/platforms/python/guides/logging) integration, so most of the examples there will work with loguru.
+
+## Install
+
+Install `sentry-sdk` from PyPI with the `loguru` extra.
+
+```bash
+pip install --upgrade 'sentry-sdk[loguru]'
+```
+
+## Configure
+
+```python
+import sentry_sdk
+from sentry_sdk.integrations.loguru import LoguruIntegration
+
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    integrations=[
+        LoguruIntegration(),
+    ],
+)
+```
+
+## Behavior
+
+By default log records with a level of `INFO` or higher will be added as breadcrumbs to Sentry events. For log records with a level of `ERROR` or higher a Sentry issue will be created:
```

Comment:
```suggestion
By default, logs with a level of `INFO` or higher will be added as breadcrumbs to Sentry events. Sentry issue will be created for logs with a level of `ERROR` or higher: 
```

Reply:
Fixed in 63f4e09e46f0ad7eb6fa2f46769f376780289f68. Thank you!